macosx/test.sh

```diff
@@ -2,12 +2,17 @@
 
 # Test to see if we have a valid TeX installation
 
-TEXMFLOCAL=`/usr/texbin/kpsewhich -var-value TEXMFLOCAL`
-
+TEXMFLOCAL=`kpsewhich -var-value TEXMFLOCAL`
```

Comment:
I don't know if you do, but you might need to handle paths in TEXMFLOCAL here also.